src/pages/BusRoutePage/components/SelectDropdown/index.tsx

```diff
@@ -7,39 +7,48 @@ import { useEscapeKeyDown } from 'utils/hooks/ui/useEscapeKeyDown';
 import { useOutsideClick } from 'utils/hooks/ui/useOutsideClick';
 import styles from './SelectDropdown.module.scss';
 
-interface ValueType {
-  date?: number;
-  hour?: number;
-  minute?: number;
-  enable: boolean;
+interface Option {
   label: string;
+  value: number;
 }
 
 interface SelectDropdownProps {
-  type: 'left' | 'middle' | 'right';
-  values: ValueType[];
-  selectedValue: number;
-  setSelectedValue: (value: number) => void;
+  type: 'dayOfMonth' | 'hour' | 'minute';
+  options: Option[];
```

Comment:
이런식으로 배열을 많이 표현하는데 아래와 같은 표현은 왜 잘 안사용하게 될까요?
아래글도 한번 읽어봤으면 좋겠어요~!

https://tkdodo.eu/blog/array-types-in-type-script

```suggestion
  options: Array<Option>;
```

Reply:
이러한 논의가 있었군요! Array<>를 사용하는 것의 근거에 설득당했습니다.☺️